lte/gateway/c/oai/tasks/s1ap/s1ap_mme_handlers.c

```diff
@@ -2536,24 +2966,22 @@ int s1ap_handle_paging_request(
   }
   const paging_tai_list_t* p_tai_list = paging_request->paging_tai_list;
   for (idx = 0; idx < enb_array->num_elements; idx++) {
-    bstring paging_msg_buffer = blk2bstr(buffer, length);
-    enb_ref_p                 = (enb_description_t*) enb_array->elements[idx];
+    enb_ref_p = (enb_description_t*) enb_array->elements[idx];
     if (enb_ref_p->s1_state == S1AP_READY) {
       supported_ta_list_t* enb_ta_list = &enb_ref_p->supported_ta_list;
 
       if ((is_tai_found = s1ap_paging_compare_ta_lists(
                enb_ta_list, p_tai_list, paging_request->tai_list_count))) {
-        rc = s1ap_mme_itti_send_sctp_request(
+        bstring paging_msg_buffer = blk2bstr(buffer_p, length);
+        rc                        = s1ap_mme_itti_send_sctp_request(
             &paging_msg_buffer, enb_ref_p->sctp_assoc_id,
             0,   // Stream id 0 for non UE related
                  // S1AP message
             0);  // mme_ue_s1ap_id 0 because UE in idle
       }
     }
   }
-  free_wrapper((void**) &enb_array->elements);
```

Comment:
Need to free enb_array->elements and enb_array here as they are calloced in `hashtable_ts_get_elements`

Reply:
Resolved

Reply:
@Vyas20 is this resolved? Do we not need to free enb_array too?